publication-commons/src/main/java/no/unit/nva/publication/external/services/PersonApiClient.java

```diff
@@ -0,0 +1,111 @@
+package no.unit.nva.publication.external.services;
+
+import static com.google.common.net.HttpHeaders.ACCEPT;
+import static no.unit.nva.publication.PublicationServiceConfig.API_HOST;
+import static no.unit.nva.publication.PublicationServiceConfig.dtoObjectMapper;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.net.MediaType;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.net.http.HttpResponse.BodyHandlers;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.apigateway.exceptions.BadGatewayException;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.paths.UriWrapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.zalando.problem.Problem;
+import org.zalando.problem.Status;
+
+public class PersonApiClient {
+
+    public static final HttpClient HTTP_CLIENT = HttpClient.newBuilder().build();
+    private static final Logger logger = LoggerFactory.getLogger(PersonApiClient.class);
+    public static final String ERROR_WITH_PERSON_SERVICE_COMMINUCATION = "Error communicating with Person service";
+    public static final String PERSON_SERVICE_RESPONSE = "Person Service response:";
+    public static final String USER_AFFILIATIONS_FIELD = "/orgunitids";
+    public static final String PATH_TO_PESON_SERVICE_PROXY = "person";
+    private final HttpClient httpClient;
+
+    public PersonApiClient(HttpClient httpClient) {
+        this.httpClient = httpClient;
+    }
+
+    @JacocoGenerated
+    public PersonApiClient() {
+        this.httpClient = HTTP_CLIENT;
+    }
+
+    public List<URI> fetchAffiliationsForUser(String feideId)
```

Comment:
The only public method of this class: It returns the CristinUnit Ids for a User.